nsqd/lookup.go

```diff
@@ -129,6 +140,13 @@ func (n *NSQD) lookupLoop() {
 					break
 				}
 			}
+		case <-n.optsNotificationChan:
```

Comment:
I may be reading this wrong, but it looks like on _every_ config change, the lookupds will be disconnected. So if I want to change the verbosity, that triggers a lookupd reconnection?

Would it be possible to diff the current lookupPeers and the list in opts to only add/remove the peers necessary? That would at least keep us from triggering a total reconnect each and every time any config value changes, which seems like (if this use case gets expanded upon) something that may happen more and more frequently over time.